openpbs.spec.in

```diff
@@ -328,6 +329,9 @@ cd build
 %if %{with ptl}
 	--enable-ptl \
 %endif
+%if %{with swig}
+	--with-swig=/usr/local \
```

Comment:
I think we should not hard code `/usr/local` here in spec file and let the caller of rpmbuild define that path. You can try below:
`rpmbuild -D '_with_swig --with-swig=/usr/local' ...`
and here you can put `%{?with_swig}`
Here is the page which explains this: http://ftp.rpm.org/api/4.4.2.2/conditionalbuilds.html